activity_browser/ui/widgets/biosphere_update.py

```diff
@@ -40,16 +36,9 @@ def update_progress(self, current: int):
 
 
 class UpdateBiosphereThread(QtCore.QThread):
-    PATCHES = (
-        add_ecoinvent_33_biosphere_flows,
-        add_ecoinvent_34_biosphere_flows,
-        add_ecoinvent_35_biosphere_flows,
-        add_ecoinvent_36_biosphere_flows,
-        add_ecoinvent_37_biosphere_flows,
-        add_ecoinvent_38_biosphere_flows,
-    )
+    PATCHES = [patch for patch in dir(data) if patch.startswith('add_ecoinvent') and patch.endswith('biosphere_flows')]
     progress = Signal(int)
-
+    data = __import__('bw2io.data')
```

Comment:
If you move this to above `PATCHES = [patch for ...`, do we still need `import bw2io.data as data` above?

Reply:
True, this does remove the need to import bw2io.data, sorry and thanks